src/state-summary/state-card-input_text.html

```diff
@@ -0,0 +1,108 @@
+<link rel="import" href="../../bower_components/polymer/polymer-element.html">
+
+<link rel="import" href="../../bower_components/iron-flex-layout/iron-flex-layout-classes.html">
+
+<link rel="import" href="../../bower_components/paper-input/paper-input.html">
+
+<link rel="import" href="../components/entity/state-info.html">
+
+<link rel="import" href="../util/hass-util.html">
+
+<dom-module id="state-card-input_text">
+  <template>
+    <style is="custom-style" include="iron-flex iron-flex-alignment"></style>
+    <style>
+      paper-input {
+        margin-left: 16px;
+      }
+    </style>
+
+    <div class='horizontal justified layout'>
+      <state-info state-obj="[[stateObj]]" in-dialog='[[inDialog]]'></state-info>
+      <paper-input
+        minlength='[[min]]' maxlength='[[max]]' value='{{value}}' pin
+        disabled='[[disabled]]' auto-validate pattern='[[pattern]]'
+        on-change='selectedValueChanged' on-tap='stopPropagation'>
+      </paper-input>
+    </div>
+  </template>
+</dom-module>
+
+<script>
+class StateCardInputText extends Polymer.Element {
+
+  static get is() { return 'state-card-input_text'; }
+
+  static get properties() {
+    return {
+      hass: Object,
+
+      inDialog: {
+        type: Boolean,
+        value: false,
+      },
+
+      stateObj: {
+        type: Object,
+        observer: 'stateObjectChanged',
+      },
+
+      min: {
+        type: Number,
+        value: 0,
+      },
+
+      max: {
+        type: Number,
+        value: 100,
+      },
+
+      pattern: {
+        type: String,
+        value: '.*',
+      },
+
+      disabled: {
+        type: Boolean,
+        value: false,
+      },
+
+      value: {
+        type: String,
+      }
+
+    };
+  }
+
+  static get observers() {
+    return [
+    ];
+  }
+
+  stateObjectChanged(newVal) {
+    this.min = Number(newVal.attributes.min);
+    this.max = Number(newVal.attributes.max);
+    if (newVal.attributes.pattern != null) {
+      this.pattern = String(newVal.attributes.pattern);
+    }
+    this.disabled = Boolean(newVal.attributes.disabled);
```

Comment:
This is not possible?

Reply:
What's not possible? I've tested this, even changed the disabled state using the developer tools / states screen in HASS and had it reflect as expected (when disabled, the input text box is greyed out but still reflects changes to the value, so automations / API can still affect it just not the user via UI).

Reply:
On further testing, it seems that I can set a value after changing `disabled = false` via tools / states but then it resets to `disabled = true` afterwards, if that is what was configured in the configuration. I've tried ensuring there's a setter for the disabled property on the component, but it still resets.